plugins/cilium-cni/chaining/flannel/flannel.go

```diff
@@ -141,6 +141,14 @@ func (f *flannelChainer) Add(ctx context.Context, pluginCtx chainingapi.PluginCo
 	return &cniTypesVer.Result{}, nil
 }
 
+func (f *flannelChainer) ImplementsDelete() bool {
```

Comment:
Why doesn't flannel support delete?

Reply:
Because there was no flannel specific delete code in the existing CNI plugin. This PR is not fixing up flannel. It is enabling chaining plugins to provide their own delete logic. We can convert flannel to be a fully native chaining plugin on top of this. I'm keeping the PR specific to enabling chaining for AWS-CNI.

Reply:
@tgraf maybe I'm missing something here. What's the purpose of this function? What do you mean by "enabling chaining for AWS-CNI"?

Reply:
The function indicates whether a plugin has its own delete logic that should be called or whether the existing CNI DEL logic should be invoked. The new `aws-cni` chaining plugin has delete functionality, the existing `portmap` and `flannel` do not have own delete logic and continue to operate as before.

Reply:
Discussed offline